proposals/lip-0068.md

```diff
@@ -156,10 +156,7 @@ Consider a binary message `trsMsg`, corresponding to a serialized transaction. T
 
 ```python
 def decodeTransaction(trsMsg: bytes) -> Transaction:
-    trsData = decode(transactionSchema,trsMsg)
-    paramsSchema = JSON schema corresponding to (trsData.module, trsData.command) pair
-    trsData.params = decode(paramsSchema,trsData.params)
-    return trsData
+    return decode(transactionSchema, trsMsg)
```

Comment:
This function is not used anywhere. We could remove it.

Reply:
I would keep it. We actually use `encodeTransaction` so I would just keep the reverse function for consistency.